catalog/spatial/registry/registry-schema-bindings/pom.xml

```diff
@@ -154,29 +184,33 @@
                         </goals>
                         <configuration>
                             <haltOnFailure>true</haltOnFailure>
+                            <includes>
```

Comment:
The includes is needed because of the generated classes. Without the includes coverage is in the 20's. Changed to include the package instead of the file names.